app/controllers/api/base_controller.rb

```diff
@@ -9,6 +9,57 @@ class BaseController < ActionController::API
     rescue_from ActionController::BadRequest, with: :bad_request_response
     rescue_from ArgumentError, with: :bad_request_response
 
+    def append_info_to_payload(payload)
+      super
+      payload[:current_user_class] = current_lead_provider&.class&.name
+      payload[:current_user_id] = current_lead_provider&.id
+      payload[:current_user_name] = current_lead_provider&.name
+
+      payload[:request_headers] = request.env.slice(
+        "CONTENT_TYPE",
+        "GATEWAY_INTERFACE",
+        "HTTP_ACCEPT",
+        "HTTP_ACCEPT_ENCODING",
+        "HTTP_AUTHORIZATION",
```

Comment:
Maybe we better off not including this as I saw: `"HTTP_AUTHORIZATION":"Bearer ambition-token"` in my tests on Logit?

Reply:
I'll remove it